nilearn/datasets.py

```diff
@@ -1539,25 +1538,61 @@ def load_harvard_oxford(atlas_name,
     regions: nibabel.Nifti1Image
         regions definition, as a label image.
     """
-    if atlas_name not in ("cort-maxprob-thr0-1mm", "cort-maxprob-thr0-2mm",
-                          "cort-maxprob-thr25-1mm", "cort-maxprob-thr25-2mm",
-                          "cort-maxprob-thr50-1mm", "cort-maxprob-thr50-2mm",
-                          "sub-maxprob-thr0-1mm", "sub-maxprob-thr0-2mm",
-                          "sub-maxprob-thr25-1mm", "sub-maxprob-thr25-2mm",
-                          "sub-maxprob-thr50-1mm", "sub-maxprob-thr50-2mm",
-                          "cort-prob-1mm", "cort-prob-2mm",
-                          "sub-prob-1mm", "sub-prob-2mm"
-                          ):
+    atlas_items = ("cort-maxprob-thr0-1mm", "cort-maxprob-thr0-2mm",
+                   "cort-maxprob-thr25-1mm", "cort-maxprob-thr25-2mm",
+                   "cort-maxprob-thr50-1mm", "cort-maxprob-thr50-2mm",
+                   "sub-maxprob-thr0-1mm", "sub-maxprob-thr0-2mm",
+                   "sub-maxprob-thr25-1mm", "sub-maxprob-thr25-2mm",
+                   "sub-maxprob-thr50-1mm", "sub-maxprob-thr50-2mm",
+                   "cort-prob-1mm", "cort-prob-2mm",
+                   "sub-prob-1mm", "sub-prob-2mm")
+    if atlas_name not in atlas_items:
```

Comment:
I think it would be better for the user to output the atlas items so the user knows the option.